### PR TITLE
Add automatic deployment scripts

### PR DESCRIPTION
**Pull request by Gentux**
This PR contains a new directory to handle automatic deployments.

First part consist of ansible script.
Second one will help to deploy several pull request on a single machine.

You can find more information on _deployments/README.md_ file

### CODE REVIEW

File: deployments/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ansible-galaxy install franklinkim.docker
+ansible-galaxy install franklinkim.docker-compose
+```
+
+> Note: You may need to have root privilege to install those dependencies. You
+> won't need it to run deployment.
+
+### Configure
+
+Configure where *ansible* will deploy your installation by modifying
```
Comment (dynamiccast) [inline deployments/README.md]: Either:
by modifying **the** ansible_hosts file.
or
by modifying ansible_hosts

File: deployments/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+*ansible_hosts* file.
+
+```
+[nanocloud]
+api ansible_host=127.0.0.1
+```
+
+In this file, you can specify several server and give name to them to deploy
```
Comment (dynamiccast) [inline deployments/README.md]: - several server**s**
- multiple instance**s**
- nanocoud -> Nanocloud

File: deployments/README.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+```
+[nanocloud]
+api-instance1 ansible_host=10.0.0.2
+api-customerX ansible_host=10.0.0.3
+```
+
+You can change hosts username to use un *nanocloud.yml*.
+
+Use the file *deployments/roles/nanocloud/files/nanocloud/config.env* to
+override configuration variable like you should normally do with *config.env*
```
Comment (dynamiccast) [inline deployments/README.md]: with **the** config.env file.

File: deployments/README.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+```
+[nanocloud]
+api-instance1 ansible_host=10.0.0.2
+api-customerX ansible_host=10.0.0.3
+```
+
+You can change hosts username to use un *nanocloud.yml*.
+
+Use the file *deployments/roles/nanocloud/files/nanocloud/config.env* to
```
Comment (dynamiccast) [inline deployments/README.md]: Should not the be backquoted ?

File: deployments/README.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+In this file, you can specify several server and give name to them to deploy
+multiple instance of nanocoud:
+
+```
+[nanocloud]
+api-instance1 ansible_host=10.0.0.2
+api-customerX ansible_host=10.0.0.3
+```
+
+You can change hosts username to use un *nanocloud.yml*.
```
Comment (dynamiccast) [inline deployments/README.md]: I did not understand that sentence.

File: deployments/README.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+Run Playbook
+
+```
+ansible-playbook nanocloud.yml
+```
+
+## Run multiple instance on a same host
+
+Script *deploy-pr.sh* is here to help you deploy a specific Github Pull Request
+on any environement without closing another instance.
```
Comment (dynamiccast) [inline deployments/README.md]: on any environement -> on any environment. 
Script -> The script
without closing another instance -> in parallel

File: deployments/README.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+You can deploy several PR in a single machine. Each deployment will have its
```
Comment (dynamiccast) [inline deployments/README.md]: in a single machine -> on a single machine

File: deployments/README.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+own *database*, *frontend*, *backend* and *guacamole-client*.
+
+To use it, run the following command:
+
+```
+./deployments/deploy-pr.sh ${PR_NUMBER}
+```
+
+And if the remote is note called *origin* you can specify another name:
```
Comment (dynamiccast) [inline deployments/README.md]: note -> not

**Comment by Gentux** [comment]
Replaced by #362

**Pull request closed without merging**